lib/jira/client/index.js

```diff
@@ -62,62 +59,33 @@ module.exports = async (id, installationId, jiraHost) => {
         return text.match(jiraIssueRegex)
       },
       comments: {
-        // eslint-disable-next-line camelcase
-        getForIssue: (issue_id) => instance.get(`/rest/api/latest/issue/:issue_id/comment`, {
-          fields: {
-            issue_id
-          }
-        }),
-        // eslint-disable-next-line camelcase
-        addForIssue: (issue_id, payload) => instance.post(`/rest/api/latest/issue/:issue_id/comment`, payload, {
-          fields: {
-            issue_id
-          }
-        })
+        getForIssue: (issueId) => instance.get(`/rest/api/latest/issue/${qs.escape(issueId)}/comment`),
+        addForIssue: (issueId, payload) => instance.post(`/rest/api/latest/issue/:${qs.escape(issueId)}/comment`, payload)
       },
       transitions: {
-        // eslint-disable-next-line camelcase
-        getForIssue: (issue_id) => instance.get(`/rest/api/latest/issue/:issue_id/transitions`, {
-          fields: {
-            issue_id
-          }
-        }),
-        // eslint-disable-next-line camelcase
-        updateForIssue: (issue_id, transition_id) => instance.post(`/rest/api/latest/issue/:issue_id/transitions`, {
-          transition: {
-            id: transition_id
-          }
-        }, {
-          fields: {
-            issue_id
-          }
-        })
+        getForIssue: (issueId) => instance.get(`/rest/api/latest/issue/${qs.escape(issueId)}/transitions`),
+        updateForIssue: (issueId, transitionId) => {
+          return instance.post(`/rest/api/latest/issue/${qs.escape(issueId)}/transitions`, { transition: { id: transitionId } })
```

Comment:
I am not sure I like the idea of performing string interpolation and sanitization at the same time. 
IMO we should either escape before interpolating or (better IMO) let the `instance` object receive an additional parameter that contains the strings to interpolate. This might be a lot of work and maybe it's not worth it...

It seems to be the common approach to have some kind of format-string and arguments, then sanitize the arguments and interpolate them. Just thinking out loud here.